Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -24,6 +25,17 @@
 
 public class MainActivity extends Activity {
 	Branch branch;
+
+    public enum SESSION_MANAGEMENT_MODE {
```

Comment:
In our iOS SDK, we have a separate BNCConfig.h file which hosts come config settings. Due to my laziness, I didn't do the same thing on the Android side. Now I think it makes sense to add it there too, and it'd be a perfect place to host this session management mode thing. We don't use BNC file prefix on the android side, so could name it BranchConfig.java.